sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogDetailLevel.java

```diff
@@ -32,7 +32,12 @@
     /**
      * Logs everything in HEADERS and BODY.
      */
-    BODY_AND_HEADERS;
+    BODY_AND_HEADERS,
+
+    /**
+     * Logs everything in QUERY PARAMS.
+     */
+    QUERY_PARAMS;
```

Comment:
This doesn't quite make sense - if the user opts in for QUERY_PARAMS, that means that they can't also get BODY or BODY_AND_HEADERS.

Reply:
so should we keep the requirement to log query params on only BODY or BODY_AND_HEADERS or both?

Reply:
`BASIC` indicates log url (that includes query parameter) and skip headers & body. Wondering why we need `QUERY_PARAMS` enum option for a specific part of url, Does guideline enforce it?

Reply:
I don't think the guideline enforces to have the `QUERY_PARAMS` enum. It can be removed, but still, not sure if we need `BODY_AND_HEADERS` with it.

Reply:
Yes, remove `QUERY_PARAMS`. Leave the rest as-is I would think.